.github/ISSUE_TEMPLATE/bug-report.md

```diff
@@ -0,0 +1,46 @@
+---
+name: Bug Report
+about: Report a bug
+title: ''
+labels: ''
+assignees: ''
+
+---
+
+## Expected Behavior
+
+_Describe what you expected to happen._
+
+## Actual behavior
+
+_Describe what actually happened._
+
+## Steps to reproduce the issue
+
+_List the steps to reproduce the issue._
+
+_1. Step 1_
```

Comment:
When rendered, this ("1. Step 1...") looks redundant...

Reply:
Removed the steps.